### ISSUE TITLE
Check cache growth

### ISSUE THREAD

**Issue by Flohack74** [ubports/morph-browser#356]
Does anyone know if Morph Browser does cache management, means to delete cache files that are too old? While moving to my new device I found 1000s of cache files, and I doubt this is the case. Rather it seems cache grows unrestricted:
![grafik](https://user-images.githubusercontent.com/16186470/88489775-d55e6a00-cf96-11ea-85e7-74385dfdfcd7.png)

**Comment by Flohack74** [comment]
Its even worse, every webapp has again a Cache2 folder and does the same. I will tally the amount of cache entries I have on my device which I used now for quite some years.

**Comment by balcy** [comment]
I think currently there is no automatic deletion of the cache, at least not in morph-browser / webapp-container

**Comment by balcy** [comment]
https://doc.qt.io/qt-5/qml-qtwebengine-webengineprofile.html#httpCacheMaximumSize-prop might be a possibility, but if may be device dependent how much is the max. cache e.g. for webapps.

**Comment by Flohack74** [comment]
Cool! We could try to set this to some value. I wonder what the value means ;)
kB MB etc....

**Comment by balcy** [comment]
in the corresponding https://doc.qt.io/qt-5.15/qwebengineprofile.html#httpCacheMaximumSize doc they say it's bytes.
We could make it a command-line parameter for webapp-container (default cache limit chosen by the developer, default 0 as now), but the user could change it in settings.
And for morph in settings.

**Comment by Flohack74** [comment]
Yeah at least to see what happens :) Can I do this locally only for testing?
